aspnetcore/blazor/state-management.md

```diff
@@ -240,38 +240,20 @@ Prerendering might be useful for other pages that don't use `localStorage` or `s
 ```cshtml
 @using Microsoft.AspNetCore.ProtectedBrowserStorage
 @inject ProtectedLocalStorage ProtectedLocalStore
-@inject IComponentContext ComponentContext
 
 ... rendering code goes here ...
 
 @code {
     private int? currentCount;
-    private bool isWaitingForConnection;
-
-    protected override async Task OnInitializedAsync()
-    {
-        if (ComponentContext.IsConnected)
```

Comment:
This hurts. I didn't realize we went to this level of optimization in the docs. We need to bring back a way to detect prerendering in 3.1.

@SteveSandersonMS @danroth27